crates/bevy_input/src/keyboard.rs

```diff
@@ -30,6 +30,16 @@ pub struct KeyboardInput {
     pub state: ButtonState,
 }
 
+impl From<&winit::event::KeyboardInput> for KeyboardInput {
```

Comment:
```suggestion
impl From<winit::event::KeyboardInput> for KeyboardInput {
```

It seems weird to do the conversion from & here and nowhere else.